### PR TITLE
Document global turbo package inference behavior

### PR DESCRIPTION
**Pull request by gsoltis**
- adds package inference to the docs on filtering workspaces
 - adds package inference to the `dev` topic in the handbook, which AFAICT is where we introduce the concept of filtering.

### CODE REVIEW

**Comment by gsoltis** [comment]
Marked as on hold because we need `1.7.1` released first

File: docs/pages/repo/docs/handbook/dev.mdx [hunk 1]
```diff
@@ -95,13 +95,22 @@ This means that users of your `dev` task _don't need to worry about codegen or m
 
 ## Running `dev` only in certain workspaces
 
-To run a `dev` task in only certain workspaces, you should use the [`--filter` syntax](/repo/docs/core-concepts/monorepos/filtering). For example:
+Let's assume you want to run the `dev` task in the `docs` workspace, located at `<root>/apps/docs`.
+With a global install of `turbo`, e.g. via `npm install --global turbo`, you can do
```
Comment (chris-olszewski) [inline docs/pages/repo/docs/handbook/dev.mdx:99]: Should we use the `selected-pkg-manager` tab here to provide all of the possible install commands?

**Review by chris-olszewski — APPROVED** [review]
Only suggested change is providing multiple install instructions for global turbo

File: docs/pages/repo/docs/handbook/dev.mdx (hunk 1 above)
Comment (gsoltis) [reply docs/pages/repo/docs/handbook/dev.mdx:99]: Good idea, I'll look into it!

Comment (gsoltis) [reply docs/pages/repo/docs/handbook/dev.mdx]: I actually decided to drop the package-manager specific part, including the reference to global turbo, in favor of just demonstrating it. I think we probably want the global turbo piece to live in a more explicit "installing turbo" section

**Comment by gsoltis** [comment]
Dropping hold, `1.7.1` is live.

**Pull request merged**